controls/6_1_system_file_permissions.rb

```diff
@@ -90,11 +90,19 @@
       it { should_not be_writable.by 'other' }
       it { should_not be_executable.by 'other' }
       its(:uid) { should cmp 0 }
-      its(:gid) { should cmp 0 }
       its(:sticky) { should equal false }
       its(:suid) { should equal false }
       its(:sgid) { should equal false }
     end
+    if os.debian?
```

Comment:
I think we can make the test more readable if we do the following instead:

```
expected_gid = 0
expected_gid = 42 if os.debian?
shadow_files.each do |f|
     describe file(f) do
       ...
       its(:gid) { should cmp expected_gid }
```

Reply:
Honestly, I'm not very good at Ruby (I'm more a Python guy), so any advice here is appreciated! I'll make the change and push it.